src/LibraryManager/Helpers/LoggingHelper.cs

```diff
@@ -0,0 +1,211 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.Web.LibraryManager.Contracts;
+
+namespace Microsoft.Web.LibraryManager.Helpers
+{
+    /// <summary>
+    /// Helper class for getting strings for output to CLI and VS
+    /// </summary>
+    public static class LoggingHelper
```

Comment:
Hypothetical:  Instead of having a nebulously named Helper namespace and class, could we instead have this be Microsoft.Web.LibraryManager.Logging.LogMessageGenerator?

Reply:
> `Microsoft.Web.LibraryManager.Logging.LogMessageGenerator` 

I like this, however, this won't match the directory structure we currently have. Are you suggesting we create a `Logging` directory (and potentially others for single classes)?

Reply:
Yes, I'm proposing naming things *well* and putting them where they should belong, even if it is the only type in the namespace.